apps/mark-scan/frontend/src/pages/contest_screen.tsx

```diff
@@ -27,7 +27,7 @@ export function ContestScreen(): JSX.Element {
     React.useContext(BallotContext);
 
   const isPathDeviceConnected = Boolean(
```

Comment:
My bad, I have an old typo here: `Path` instead of `Pat` (duplicated in `voter_flow.tsx`) 😅 -- could you fix while you're here?